### PR TITLE
[DEV-829]: Improve scripts loading

### PR DESCRIPTION
**Pull request by MikeAtUqido**
#### List of Changes

 * Use next's `Script` component to load scripts

#### Motivation and Context

Using the html `<script>` tag was discouraged by Next

#### How Has This Been Tested?

 - [x] Manually

#### Types of changes

- [x] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/nextjs-website/src/components/atoms/CtaSlide/CtaSlide.tsx [hunk 1]
```diff
@@ -67,6 +67,7 @@ const CtaSlide = ({
           src={backgroundImage}
           alt={title}
           fill={true}
+          priority
```
Comment (marcobottaro) [inline apps/nextjs-website/src/components/atoms/CtaSlide/CtaSlide.tsx:70]: Why did you add the priority prop to the Hero Image?

Comment (MikeAtUqido) [reply apps/nextjs-website/src/components/atoms/CtaSlide/CtaSlide.tsx:70]: I addded it because there was a warning relative to it, and the `priority` resolves the issue, but we can remove it if we don't need it.
Screenshot 2024-01-09 alle 14 46 19

Comment (marcobottaro) [reply apps/nextjs-website/src/components/atoms/CtaSlide/CtaSlide.tsx:70]: Please remove it from this PR, add a task to the board, and open a new PR

**Review by marcobottaro — CHANGES_REQUESTED** [review]
Please remove the [hero image bug fix](https://github.com/pagopa/developer-portal/pull/542/files#r1446331722) from this PR

**Review by marcobottaro — APPROVED** [review]
